### DISCUSSION TITLE
What happened to compile.ops.Rebroadcast?

### DISCUSSION THREAD

**Discussion by habibi1195** [aesara-devs/aesara#1516] [Q&A] [answered]
I had an old project that used Theano. I had a piece of code that relied on theano.tensor.compile.ops.Rebroadcast. I see that this isn't available in Aesara. What happened to it, and is there something else I can use to make my biases in forward propagation to be broadcastable in axis 1 but not in axis 0?

Thank you in advance.

**Comment by brandonwillard** [comment, marked as answer]
See https://github.com/aesara-devs/aesara/pull/915#issue-1208263270.

**Reply by brandonwillard** [reply]
Also, it's worth seeing if your old code works without `Rebroadcast`.  A lot of updates were made to fundamental `Op`s that removed the need for it entirely.
